tests/TestCase/View/Helper/FormHelperTest.php

```diff
@@ -6833,11 +6833,11 @@ public function testButton(): void
         $expected = ['button' => ['type' => 'submit'], 'Hi', '/button'];
         $this->assertHtml($expected, $result);
 
-        $result = $this->Form->button('Clear Form >', ['type' => 'reset', 'escapeTitle' => false]);
+        $result = $this->Form->button('Clear Form >', ['type' => 'reset', 'escape' => false]);
```

Comment:
While I haven't found a generic rector rule that would handle this, there is a similar kind of rule in the symfony set that we could probably base a rector for this on https://getrector.com/rule-detail/option-name-rector